### PR TITLE
Update chaos engineering term

### PR DESCRIPTION
**Pull request by electrocucaracha**
### Describe your changes
This change synchronizes the changes made on the `chaos engineering` term to the Spanish version.

### Related issue number or link (ex: `resolves #issue-number`)
NA

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by electrocucaracha** [comment]
@raelga @92nqb pueden revisar esta actualización?

File: content/es/chaos-engineering.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+---
+
+## ¿Qué es?
+
+Ingeniería del Caos o CE (Chaos Engineering) es la disciplina de experimentación sobre un [Sistema Distribuido](/distributed_systems/) en producción
+para construir confianza en la capacidad del sistema para soportar condiciones inesperadas y turbulentas.
+
+## Problema que aborda
+
+Las prácticas de [SRE](/site_reliability_engineering/) y [DevOps](/es/devops/) se centran
```
Comment (92nqb) [inline content/es/chaos-engineering.md:15]:
```suggestion
Las prácticas de [SRE](/site_reliability_engineering/) y [DevOps](/es/devops/) se centran en
```

File: content/es/chaos-engineering.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+técnicas para el incremento de la resiliencia del producto y de la [confiabilidad](/es/reliability/).
+La capacidad del sistema para tolerar fallos al mismo tiempo que aseguran una calidad de servicio adecuado
+suele ser un típico requerimiento de desarrollo de software.
+Existen muchos aspectos involucrados al momento de la indisponibilidad de una aplicación,
+como la infraestructura, la plataforma o otras partes del ecosistema de las aplicaciones basadas en ([microservicios](/microservices/)).
+La alta frecuencia de despliegues de funcionalidades hacia el ambiente productivo puede
+aumentar la posibilidad de un incidente crítico o estar fuera de línea
+- generando consecuencias considerables para el negocio.
```
Comment (92nqb) [inline content/es/chaos-engineering.md:23]:
```suggestion
generando consecuencias considerables para el negocio.
```
No se si en español se usa este simbolo -. En chino han puesto una ,

File: content/es/chaos-engineering.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+## Problema que aborda
+
+Las prácticas de [SRE](/site_reliability_engineering/) y [DevOps](/es/devops/) se centran
+técnicas para el incremento de la resiliencia del producto y de la [confiabilidad](/es/reliability/).
+La capacidad del sistema para tolerar fallos al mismo tiempo que aseguran una calidad de servicio adecuado
+suele ser un típico requerimiento de desarrollo de software.
+Existen muchos aspectos involucrados al momento de la indisponibilidad de una aplicación,
+como la infraestructura, la plataforma o otras partes del ecosistema de las aplicaciones basadas en ([microservicios](/microservices/)).
+La alta frecuencia de despliegues de funcionalidades hacia el ambiente productivo puede
+aumentar la posibilidad de un incidente crítico o estar fuera de línea
```
Comment (92nqb) [inline content/es/chaos-engineering.md:22]:
```suggestion
aumentar la posibilidad de un incidente crítico o estar fuera de línea,
```

File: content/es/chaos-engineering.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+- generando consecuencias considerables para el negocio.
+
+## ¿Cómo ayuda?
+
+La Ingeniería del Caos es una técnica para cumplir los requerimientos de resiliencia.
+Esta disciplina ayuda a obtener una infraestructura, plataforma y aplicación resistente ante fallos.
+Los ingenieros utilizan los experimentos de caos de forma proactiva inyectando fallas aleatorias
+para verificar que la infraestructura, plataforma y aplicación puedan auto recuperarse y que la falla no sea de notable impacto para los usuarios finales.
+Los experimentos de caos buscan descubrir puntos ciegos
+(por ejemplo, monitoreo o técnicas de autoescalamiento) y mejorar la comunicación entre los equipos durante incidentes críticos.
```
Comment (92nqb) [inline content/es/chaos-engineering.md:32]:
```suggestion
(por ejemplo, monitoreo o técnicas de autoescalado) y mejorar la comunicación entre los equipos durante incidentes críticos.
```

**Review by 92nqb — COMMENTED** [review]
I've added some comments

**Review by raelga — APPROVED** [review]

**Pull request merged**